atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -36,7 +38,13 @@ def __init__(self, tasker, workflow, hub, target=None):
         """
         # call parent constructor
         super(BumpReleasePlugin, self).__init__(tasker, workflow)
-        self.xmlrpc = create_koji_session(hub)
+        koji_auth_info = {
+            'proxyuser': koji_proxyuser,
+            'ssl_certs_dir': koji_ssl_certs_dir,
+            'krb_principal': koji_krb_principal,
+            'krb_keytab': koji_krb_keytab,
+        }
+        self.xmlrpc = create_koji_session(hub, koji_auth_info)
```

Comment:
Right... but this plugin doesn't do *anything* that could require authentication. It's done anonymously. Any Koji installation that doesn't allow that is broken.